doc/stdenv/meta.chapter.md

```diff
@@ -22,18 +22,32 @@ Meta-attributes are not passed to the builder of the package. Thus, a change to
 
 ## Standard meta-attributes {#sec-standard-meta-attributes}
 
+If the package is to submitted to Nixpkgs, also check out
+[requirements of meta attributes](https://github.com/NixOS/nixpkgs/tree/master/pkgs#meta-attributes)
+in the contributing documentation.
```

Comment:
The current wording sounds a bit off to my ear, I could be wrong, yet this sounds more correct to me:
```suggestion
If the package is to be submitted to Nixpkgs, please check out the
[requirements for meta attributes](https://github.com/NixOS/nixpkgs/tree/master/pkgs#meta-attributes)
in the contributing documentation.
```